pages/content/amp-dev/documentation/guides-and-tutorials/start/create/actions_events.md

```diff
@@ -0,0 +1,42 @@
+---
+$title: Actions and events
+$order: 5
+description: 'AMP embraces interactive user experiences.'
+author: crystalonscript
+---
+
+AMP embraces interactive user experiences. But, for performance and user experience guarantees, it’s just done a bit differently than non-AMP pages.
+
+AMP uses the `on` attribute to install event handlers on elements. Like attributes, some events and actions are available to common elements while others are special to certain components. For our first page, we’ll register a basic event, a user’s click, then respond with a popular action, hide.
+
+Add a button to your page, and give it the `on` attribute:
+
+```html
+<button on="">
+    Goodbye AMPHTML World!
+</button>
+```
+
+The first thing we’ll define in the `on` attribute’s value is the event we’re listening for. For a user’s click via mouse on desktop or mobile touch, AMP uses the event value `tap`.
+
+```html
+<button on="tap">
+```
+
+Then, we add a colon character, then define the `id` of the target we want our action to have an effect on. We’ll hide our `<h1>` element, so let's add “hello” after the colon character.
```

Comment:
This is the first we're hearing of the `h1` element, should it be referenced earlier? 

also, suggestion for clarity: 
```suggestion
Then, we add a colon character, then define the `id` of the target we want our action to have an effect on. We’ll hide our `<h1 id="hello">` element, so let's add “hello” after the colon character.
```